gammapy/background/tests/test_models.py

```diff
@@ -21,30 +21,27 @@
 class TestGaussianBand2D:
     def setup(self):
         table = Table()
-        table['x'] = [-30, -10, 10, 20]
-        table['amplitude'] = [0, 1, 10, 0]
-        table['mean'] = [-1, 0, 1, 0]
-        table['stddev'] = [0.4, 0.5, 0.3, 1.0]
+        table['GLON'] = [-30, -10, 10, 20] * u.deg
+        table['Surface_Brightness'] = [0, 1, 10, 0] * u.Unit('cm-2 s-1 sr-1')
+        table['GLAT'] = [-1, 0, 1, 0] * u.deg
+        table['Width'] = [0.4, 0.5, 0.3, 1.0] * u.deg
         self.table = table
         self.model = GaussianBand2D(table)
 
     def test_evaluate(self):
         x = np.linspace(-100, 20, 5)
         y = np.linspace(-2, 2, 7)
         x, y = np.meshgrid(x, y)
-        image = self.model.evaluate(x, y)
-        assert_allclose(image.sum(), 1.223962643740966)
+        coords = SkyCoord(x, y, unit='deg', frame='galactic')
+        image = self.model.evaluate(coords)
+        desired = 1.223962643740966 * u.Unit('cm-2 s-1 sr-1')
+        assert_quantity_allclose(image.sum(), desired)
 
     def test_parvals(self):
-        par = self.model.parvals(-30)
-        assert_allclose(par['amplitude'], 0)
-        assert_allclose(par['mean'], -1)
-        assert_allclose(par['stddev'], 0.4)
-
-    def test_y_model(self):
-        model = self.model.y_model(-30)
-        assert isinstance(model, Gaussian1D)
-        assert_allclose(model.parameters, [0, -1, 0.4])
+        glon = Angle(-30, unit='deg')
+        assert_quantity_allclose(self.model.peak_brightness(glon), 0 * u.Unit('cm-2 s-1 sr-1'))
```

Comment:
Pick some position for assert where value is != 0? Would that test a bit more or doesn't it matter?